### PR TITLE
Add information on using manually labeled datasets for simulation

### PR DESCRIPTION
**Pull request by SagevdBrand**
Please let me know if you have any additions!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Systematic Review Datasets
 
-This repository provides an overview of labeled datasets used for Systematic Reviews. The
-datasets are available under an open licence and can be used for text mining and machine
+This repository provides an overview of pre-labeled datasets used for Systematic Reviews. The
```
Comment (J535D165) [inline README.md:3]: What's the difference between "pre-labeled" and "labeled"?

Comment (SagevdBrand) [reply README.md:3]: Good question, not sure! In the blogpost I portrayed it as a dataset that has been labeled previously, but actually pre-labeled is the same as labeled then?

**Review by Rensvandeschoot — APPROVED** [review]

**Comment by J535D165** [comment]
I think we should close this PR. This project is not ASReview specific and therefore it feels a bit odd to have documentation like this in there. 

ps. Yes, there are more references to ASReview introduced in this repo, but not intentionally.

**Comment by Rensvandeschoot** [comment]
I agree.

**Pull request closed without merging**